src/core/tooltips/rfc.md

```diff
@@ -0,0 +1,86 @@
+## Tooltip Service
+
+This service allows us to show tooltips in different places of our application.
+Underhood it uses `@konturio/ui-kit` tooltip component for show tooltip in theme style
+
+Tooltip:
+
+- Shows by hover-in and hides by hover-out events (after delay)
+- Not interactive
+- Can contain only text
+
+## How to use
+
+We assume that all cases of using a tooltip can be reduced to two cases
+
+### We have an element around which a tooltip should be shown
+
+In that case you need to import special components and use them
+
+```jsx
+import { Tooltip, TooltipTrigger, TooltipContent } from '~core/tooltip';
+
+function Sidebar() {
+  return (
+    <nav>
+      <Tooltip>
```

Comment:
The tooltip open status will be uncontrolled by default, it will be useful to have an `open` prop for controlled usage.

Reply:
For what case you need that?
In case you need to open/close it manually you want to use service directly as in example with map

Reply:
For example, show a tooltip on some component when hovering some another

Reply:
Sounds like broken ux : )
But if you really need that you still can do it:
```ts
triggerElement.addEventListener('pointer', (e) => {
  tooltip.show(
      ElementWhereToShow,
      'Some text'
    );
})
  ```